code/model/Learner.cpp

```diff
@@ -109,6 +115,15 @@ bool SubgoalLearner::Init (void)
 	return (-1 != i_DomainPddlId);
 }
 
+//										
+void SubgoalLearner::OnIRAnswer (int _iIndex, IRAnswer& _aAnswer)
+{
+	// pthread_mutex_lock (&mtx_WaitForSequences);
+	// TODO: do something when IRAnswer is received
+	// pthread_mutex_unlock (&mtx_WaitForSequences);
+}
+
+
```

Comment:
Not sure if this should be in subgoal policy, I think this could set a variable, than it is then seen by subgoal policy and subgoal policy then checks on that - or similar, will see